DifferenceKit.podspec

```diff
@@ -14,6 +14,8 @@ Pod::Spec.new do |spec|
   spec.requires_arc = true
   spec.default_subspecs = 'Core', 'UIExtensions'
 
+  spec.swift_version = '4.1'
```

Comment:
Can we use variable to ensure the same Swift version in the main spec and sub spec?
```ruby
swift_version = '4.1'
spec.swift_version = swift_version

subspec.swift_version = swift_version
```

Reply:
I think `spec.swift_version` can be just removed.

See https://guides.cocoapods.org/syntax/podspec.html#subspec

`On the other side, a ‘sub-specification’ inherits the value of the attributes of the parents so common values for attributes can be specified in the ancestors.`

Reply:
removed it already

Reply:
Do we need to specify the swift version for the pod if it supports both 4.1 and 4.2. I believe letting it inherit from the main project is the best way to go.

Reply:
I assume that if `swift_version` isn't  specified, CocoaPods will use main target swift version. So it will lead to compilation failures in case if only Swift 4.1 supported. This's what I had.

Better solution that CocoaPods doesn't support yet is specifying swift version range. CocoaPods has issue on this.

Reply:
I see that Swift 4.2 support was just merged into `master`. So this PR may be skipped.

Reply:
I'm only superficially undorstand the issue, can we close this PR?